templates/download/server/download.html

```diff
@@ -58,53 +61,59 @@ <h2>Ubuntu CLI cheatsheet</h2>
+    <div class="row">
       <div class="col-3 col-medium-2">
-        <h3 class="p-heading--5"><a href="/server/docs">Ubuntu Server documentation</a></h3>
-        <p>The Ubuntu Server documentation, curated by community and Ubuntu developers.</p>
+        <div class="p-section">
+          <h2 class="p-muted-heading">Resources</h2>
+        </div>
       </div>
-      <div class="col-3 col-medium-2">
-        <h3 class="p-heading--5"><a href="https://askubuntu.com/">Ask Ubuntu</a></h3>
-        <p>Stuck somewhere? Get your Ubuntu questions answered by power users.</p>
+      <div class="col-9">
+        <div class="row">
+          <div class="col-3 col-medium-2">
+            <h3 class="p-heading--5">
+              <a href="/server/docs">Ubuntu Server documentation</a>
+            </h3>
+            <p>The Ubuntu Server documentation, curated by community and Ubuntu developers.</p>
+          </div>
+          <div class="col-3 col-medium-2">
+            <h3 class="p-heading--5">
+              <a href="https://askubuntu.com/">Ask Ubuntu</a>
+            </h3>
+            <p>Stuck somewhere? Get your Ubuntu questions answered by power users.</p>
+          </div>
+          <div class="col-3 col-medium-2">
+            <h3 class="p-heading--5">
+              <a href="https://discourse.ubuntu.com/">Community Discourse</a>
+            </h3>
+            <p>Join the Ubuntu developers community and make your voice heard.</p>
+          </div>
+        </div>
       </div>
-      <div class="col-3 col-medium-2">
-        <h3 class="p-heading--5"><a href="https://discourse.ubuntu.com/">Community Discourse</a></h3>
-        <p>Join the Ubuntu developers community and make your voice heard.</p>
-      </div>  
-     </div> 
     </div>
-  </div>
-</section>
+  </section>
 
-{% include 'shared/_download-ubuntu-thank-you-pro-section.html' %}
+  {% include 'shared/_download-ubuntu-thank-you-pro-section.html' %}
 
-<section class="p-section">
-  <div class="p-section--shallow u-fixed-width">
-    <hr class="p-rule" />
-    <h2 class="p-muted-heading">More from Canonical</h2>
-  </div>
-  <div class="row">
-    <div class="col-3 col-medium-6">
-      <div class="row">
-        <div class="col-6 col-medium-2">
-          <h3>
-            <a href="https://maas.io/?_gl=1*1y3394w*_gcl_au*Mzg2MjE3ODk3LjE3MDI0NjY1NDY.&_ga=2.256850242.873815666.1709630805-805571671.1709630804">
-              {{
+  <section class="p-section">
+    <div class="p-section--shallow u-fixed-width">
+      <hr class="p-rule" />
+      <h2 class="p-muted-heading">More from Canonical</h2>
+    </div>
+    <div class="row">
+      <div class="col-3 col-medium-6">
+        <div class="row">
+          <div class="col-6 col-medium-2">
+            <h3>
+              <a href="https://maas.io/?_gl=1*1y3394w*_gcl_au*Mzg2MjE3ODk3LjE3MDI0NjY1NDY.&_ga=2.256850242.873815666.1709630805-805571671.1709630804">
+                {{
                 image (
```

Comment:
since you have formatted this file, could you please fix this function call as well, by removing the space and reformatting?